### ISSUE TITLE
Cannot install plugin on node 16

### ISSUE THREAD

**Issue by slitviachenko** [adobe/aio-cli-plugin-api-mesh#53]
### Expected Behaviour
I can install the "@adobe/aio-cli-plugin-api-mesh" plugin on node 16 using the `aio plugins install @adobe/aio-cli-plugin-api-mesh` command.

### Actual Behaviour
`aio plugins install @adobe/aio-cli-plugin-api-mesh` on node16 gives the following error message:
```
error @adobe/aio-cli-plugin-api-mesh@2.0.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.18.1"
error Found incompatible module.
Installing plugin @adobe/aio-cli-plugin-api-mesh... failed
```

### Reproduce Scenario (including but not limited to)
Try to install a plugin using the command below on node16:
```
% node --version
v16.18.1
% aio plugins install @adobe/aio-cli-plugin-api-mesh
```

#### Steps to Reproduce
See the steps above.

#### Platform and Version
`@adobe/aio-cli-plugin-api-mesh@2.0.0`
```
% node --version
v16.18.1
% npm --version
8.19.3
% aio --version
@adobe/aio-cli/9.1.1 darwin-x64 node-v16.18.1
```

#### Sample Code that illustrates the problem
This plugin has node version requirements that restrict it to node >=18.0.0, see https://github.com/adobe/aio-cli-plugin-api-mesh/blob/8a87b7c3bc5e9df56b44ecd02b77128e4a20579b/package.json#L54

#### Logs taken while reproducing problem
```
error @adobe/aio-cli-plugin-api-mesh@2.0.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.18.1"
error Found incompatible module.
Installing plugin @adobe/aio-cli-plugin-api-mesh... failed
```

**Comment by shazron** [comment]
aio-cli still supports >=14 so it's prudent if this plugin can do the same unless it's using some node-18 feature? 
https://github.com/adobe/aio-cli/blob/5bacaaf920cd338e658c5d2beaadb38ee0a158ae/package.json#L56

**Comment by jward-magento** [comment]
Fixed.

**Comment by jward-magento** [comment]
fixed in 2.1.0 release
